models/tornado_cash/tornado_cash_sources.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 sources:
   - name: tornado_cash_bnb
     freshness:
-      warn_after: { count: 12, period: hour }
+      warn_after: { count: 7, period: day }
```

Comment:
Why 7 days here and 12h below? I think these should also be 12h if it's not too much

Reply:
oh lol okay then it's fine I guess